### PR TITLE
Rename to spectrum theme

### LINKED ISSUE #69: Come up with a new name
Although "Modern Dark" works as a name, it might be worth coming up with a better name. On twitter, an Eclipse contributor mentioned that the word "Modern" can quickly get out-of-sync with changes in UI trends.

@pyvesb suggested naming the theme "Eclipse Spectrum Theme" which sounds nice to me. 

To quote his reasoning (in the context of the new logo ideas):
"_The dark background and the colored lines reminds me of those color spectrum charts used in physics, chemistry and astronomy. Hence my suggestion: "Eclipse Spectrum Theme". :)_"

Ideally, if the plugin is going to be renamed, it should occur before the marketplace release. I could always change the name after the 0.0.1 release, but it might confuse some users (I'm not sure how many downloads the plugin will get for the initial marketplace release, so it might not be a big deal)

All name ideas are welcome :)

### PR DESCRIPTION
**Pull request by AObuchow**
Fix #69

### CODE REVIEW

**Comment by AObuchow** [comment]
Some of the id's (both in the plugin.xml and in the css) are still using "Modern Dark". Also the repo needs to be renamed as well + the update site link.

File: .project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>Eclipse-Modern-Dark-Theme</name>
```
Comment (AObuchow) [inline .project:3]: Rename to Eclipse-Spectrum-Theme ?

File: com.aobuchow.themes.spectrum.feature/feature.properties [hunk 2]
```diff
@@ -0,0 +1,4 @@
+featureName=Spectrum Theme
+featureURL=https://github.com/AObuchow/Eclipse-Modern-Dark-Theme
```
Comment (AObuchow) [inline com.aobuchow.themes.spectrum.feature/feature.properties:2]: This URL needs to be updated when the github repo is renamed.

File: com.aobuchow.themes.spectrum.feature/feature.xml [hunk 3]
```diff
@@ -20,10 +20,10 @@
    </license>
 
    <plugin
-         id="com.aobuchow.themes.moderndark"
+         id="com.aobuchow.themes.spectrum"
          download-size="0"
          install-size="0"
-         version="1.0.7.qualifier"
+         version="0.0.0"
```
Comment (AObuchow) [inline com.aobuchow.themes.spectrum.feature/feature.xml:26]: I changed the feature's plugin version to 0.0.0 which is supposed to automatically resolve to the latest?

**Comment by AObuchow** [comment]
The .gitignore's also have to be updated.

**Comment by AObuchow** [comment]
> Some of the id's (both in the plugin.xml and in the css) are still using "Modern Dark". 

I think this might be too risky to do before the initial marketplace release.

I was thinking of making an “experimental” branch to work on 0.0.2, there I can rename the id’s without worry of breaking the plugin, and I can also deliver fixes to 0.0.1 

The experimental branch would also be used to add new features such as a preference page.

File: .project (hunk 1 above)
Comment (ingomohr) [reply .project:3]: Yup. :)

File: com.aobuchow.themes.spectrum.feature/feature.properties (hunk 2 above)
Comment (ingomohr) [reply com.aobuchow.themes.spectrum.feature/feature.properties:2]: Hm. Does this mean you have to move your Git repo, too? Maybe there's a way to rename Githup repos, too... - or to move the issues from one repo to another, so you can keep all the tickets and their history.

Comment (AObuchow) [reply com.aobuchow.themes.spectrum.feature/feature.properties:2]: Yes there's a feature to rename repo's thankfully :D

**Comment by AObuchow** [comment]
> > Some of the id's (both in the plugin.xml and in the css) are still using "Modern Dark". 
> 
> I think this might be too risky to do before the initial marketplace release.
> 
> I was thinking of making an “experimental” branch to work on 0.0.2, there I can rename the id’s without worry of breaking the plugin, and I can also deliver fixes to 0.0.1 
> 
> The experimental branch would also be used to add new features such as a preference page. 
> 
> 

@ingomohr this might be a good chance to try out your new github action ;)

**Comment by ingomohr** [comment]
> > > Some of the id's (both in the plugin.xml and in the css) are still using "Modern Dark". 
> > 
> > I think this might be too risky to do before the initial marketplace release.
> > 
> > I was thinking of making an “experimental” branch to work on 0.0.2, there I can rename the id’s without worry of breaking the plugin, and I can also deliver fixes to 0.0.1 
> > 
> > The experimental branch would also be used to add new features such as a preference page. 
> > 
> > 
> 
> @ingomohr this might be a good chance to try out your new github action ;)

@AObuchow You‘re the boss. 😜. I should finish it tonight (or tomorrow where my free time part of the week starts). Then we can use it right away. 😊

**Comment by AObuchow** [comment]
> > > > Some of the id's (both in the plugin.xml and in the css) are still using "Modern Dark". 
> > > 
> > > I think this might be too risky to do before the initial marketplace release.
> > > 
> > > I was thinking of making an “experimental” branch to work on 0.0.2, there I can rename the id’s without worry of breaking the plugin, and I can also deliver fixes to 0.0.1 
> > > 
> > > The experimental branch would also be used to add new features such as a preference page. 
> > > 
> > > 
> > 
> > @ingomohr this might be a good chance to try out your new github action ;)
> 
> @AObuchow You‘re the boss. 😜. I should finish it tonight (or tomorrow where my free time part of the week starts). Then we can use it right away. 😊

Awesome thank you so much @ingomohr ! No rush needed, whenever you have the time :)

**Pull request merged**